src/utils/provideLocalStorageDataClass.ts

```diff
@@ -36,7 +36,11 @@ export function provideLocalStorageDataClass(
             : items.filter((item) =>
                 Object.entries(filters).every(
                   ([filterAttribute, filterValue]) =>
-                    item[filterAttribute] === filterValue,
+                    item[filterAttribute] === filterValue ||
+                    (filterValue === 'true' &&
+                      item[filterAttribute] === true) ||
+                    (filterValue === 'false' &&
+                      item[filterAttribute] === false),
```

Comment:
* [ ] FYI Alternatively we could simplify/generalize the check with something like `item[filterAttribute]?.toString() === filterValue`

Reply:
I would like to keep this code specific for boolean attributes.